README.md

```diff
@@ -2,20 +2,18 @@
 
 JupyterLab Desktop is the cross-platform desktop application for [JupyterLab](https://github.com/jupyterlab/jupyterlab). It is the quickest and easiest way to get started with Jupyter notebooks on your personal computer, with the flexibility for advanced use cases.
 
-![JupyterLab Desktop](media/jupyterlab-desktop.png)
```

Comment:
I think this is a good tight sentence.  I like the screenshot early.

My thinking is, when I land on a repo for a piece of software, the software should sell the benefits first, then explain how to install.  If a repo explains how to install before showing me what it does, my thought is "why do I want to go through the effort of downloading this thing , when I don't know how it helps me".

It is also worthwhile to disqualify users who will never be a match for a product early.